### ISSUE TITLE
ImagePullError for authservice-0 Pod in Kubeflow v1.7 on EKS v1.28 - Image Missing from GCR

### ISSUE THREAD

**Issue by talha-naeem1** [kubeflow/manifests#2851]
### Validation Checklist

- [X] Is this a Kubeflow issue?
- [X] Are you posting in the right repository ?
- [X] Did you follow the installation guide https://github.com/kubeflow/manifests?tab=readme-ov-file ?
- [X] Is the issue report properly structured and detailed with version numbers?
- [X] Is this for Kubeflow development ?
- [ ] Would you like to work on this issue?
- [X] You can join the CNCF Slack and access our meetings at the [Kubeflow Community](https://www.kubeflow.org/docs/about/community/) website. Our channel on the CNCF Slack is here [**#kubeflow-platform**](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2).

### Version

1.8

### Describe your issue

I’m using Kubeflow v1.7 on my EKS v1.28 cluster, and it was working fine until recently. However, I’m now encountering an ImagePullError for the authservice-0 pod with the image gcr.io/arrikto/kubeflow/oidc-authservice:e236439. Additionally, I can no longer find this image on the Google Container Registry (GCR) or pull it locally.

It would be really helpful if we could resolve this issue or get any direction for its resolution.

### Steps to reproduce the issue

Deployment of v1.7 via kustomize

### Put here any screenshots or videos (optional)

![Screenshot from 2024-08-21 13-38-15](https://github.com/user-attachments/assets/48b760ae-7bdd-481c-b3a2-9af87433d948)

**Comment by juliusvonkohout** [comment]
Hello, can you take the image from Kubeflow 1.8.1?

**Comment by juliusvonkohout** [comment]
What about 
```
Images:
  - name: gcr.io/arrikto/kubeflow/oidc-authservice
    newName: docker.io/kubeflowmanifestswg/oidc-authservice
    newTag: e236439
```

But this is just a workaround. Please Upgrade to Kubeflow 1.8.1/1.9

**Comment by k2-gc** [comment]
@juliusvonkohout 
Thank you very much.

> What about
> 
> ```
> Images:
>   - name: gcr.io/arrikto/kubeflow/oidc-authservice
>     newName: docker.io/kubeflowmanifestswg/oidc-authservice
>     newTag: e236439
> ```
> 
> But this is just a workaround. Please Upgrade to Kubeflow 1.8.1/1.9

Seems it works for me!

**Comment by juliusvonkohout** [comment]
@k2-gc just be warned that we fixed a lot of stuff in 1.8.1/1.9 so running an old insecure 1.7 in enterprise environments is questionable, although it might be fine for academic use.

**Comment by talha-naeem1** [comment]
Thanks, It worked for me.
